https://github.com/unisat-wallet/extension/pull/271

Jokacar10 opened the pull request.

drgabriel11 reviewed (commented):
Stake up my portfolio

slient-coder committed:
feat: add brc2.0

fix

fix

fix

fix

slient-coder committed:
chore: bump version to 1.7.3

snyk-bot committed:
fix: package.json & yarn.lock to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-7577916
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-7577917
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-7577918
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-8720086
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-8172694

Jokacar10 committed:
Merge pull request #23 from Jokacar10/snyk-fix-1f5f146b05b3794ad21292b00d905f86

[Snyk] Security upgrade @cosmjs/stargate from 0.33.0 to 0.34.0

snyk-bot committed:
fix: package.json & yarn.lock to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-JSYAML-13961110

Jokacar10 committed:
Merge pull request #26 from Jokacar10/snyk-fix-864dece3a6ad8ffb235760c0e3a5c239

[Snyk] Security upgrade @svgr/webpack from 5.5.0 to 6.0.0

snyk-bot committed:
fix: package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-ESLINT-15102420

snyk-bot committed:
fix: package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-WEBPACK-15235959
- https://snyk.io/vuln/SNYK-JS-WEBPACK-15235969

Jokacar10 committed:
Merge pull request #37 from Jokacar10/snyk-fix-ab04b3e1e956b6320e8f6f9732806e68

[Snyk] Security upgrade eslint from 8.34.0 to 9.26.0

Jokacar10 committed:
Merge pull request #38 from Jokacar10/snyk-fix-a04b30536dd0a0197a68dccdce9a4773

[Snyk] Security upgrade webpack from 5.75.0 to 5.104.1
